app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -692,6 +686,8 @@ def parse_container_image(image, imageID)
         if stored_container_image_registry.nil?
           @data_index.store_path(
             :container_image_registry, :by_host_and_port, host_port, container_image_registry)
+          @data[:container_image_registries] ||= []
```

Comment:
@enoodle maybe we can move this initialization somewhere else?

There's an edge-case where all the images have no registry where this may remain `nil`, right? (As we'll never hit this code).